e2e/actions/l2_challenger_test.go

```diff
@@ -13,14 +15,14 @@ import (
 	"github.com/kroma-network/kroma/components/node/testlog"
 	chal "github.com/kroma-network/kroma/components/validator/challenge"
 	"github.com/kroma-network/kroma/e2e/e2eutils"
+	"github.com/kroma-network/kroma/e2e/testdata"
 )
 
-const lastValidBlockNum = uint64(6)
-
 func TestChallenger(gt *testing.T) {
 	t := NewDefaultTesting(gt)
 	dp := e2eutils.MakeDeployParams(t, defaultRollupTestParams)
 	dp.DeployConfig.FinalizationPeriodSeconds = 60 * 60 * 24
+	dp.DeployConfig.ColosseumDummyHash = common.HexToHash("0xaf01bc158f9b35867aea1517e84cf67eedc6a397c0df380b4b139eb570ddb2fc")
```

Comment:
Could you extract `0xaf01bc158f9b35867aea1517e84cf67eedc6a397c0df380b4b139eb570ddb2fc` to constant?

Reply:
done